### PR TITLE
Add chapter on Process Sets and Groups

### PR DESCRIPTION
**Pull request by rhc54**
Describe the overall concepts and how these two definitions differ.
Define APIs and attributes

Signed-off-by: Ralph Castain <rhc@open-mpi.org>

dd

Signed-off-by: Ralph Castain <rhc@open-mpi.org>

### CODE REVIEW

**Comment by rhc54** [comment]
@hjelmn Could you please review this? I have attached the pdf form of the new chapter to make for easier reading.

[ProcessSetsGroups.pdf](https://github.com/pmix/pmix-standard/files/2715771/ProcessSetsGroups.pdf)

**Comment by hppritcha** [comment]
@rhc54 This looks pretty good.  The clarification about difference between sets and groups is useful.  I had one minor comment for the opening paragraph.

File: Chap_API_Sets_Groups.tex [hunk 1]
```diff
@@ -0,0 +1,810 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Chapter: Process Sets and Groups
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\chapter{Process Sets and Groups}
+\label{chap:api_sets_groups}
+
+\ac{PMIx} supports two slightly related, but functionally different concepts known as \emph{process sets} and \emph{process groups}. This chapter describes the two definitions and how they are utilized, along with their corresponding \acp{API}.
+
```
[the remark this answers is not in the document]
Comment (rhc54) [reply Chap_API_Sets_Groups.tex]: Thanks @hppritcha  - I like your second option and will go with it!

**Comment by rhc54** [comment]
Per @hppritcha, this is good to go!

**Pull request merged**